### PR TITLE
Ensure gem dependencies are secure

### PR DESCRIPTION
**Pull request by thommay**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,6 +30,8 @@ group(:development, :test) do
 
   gem 'ruby-shadow', :platforms => :ruby unless RUBY_PLATFORM.downcase.match(/(aix|cygwin)/)
 
+  gem 'bundler-audit', git: 'https://github.com/rubysec/bundler-audit.git', ref: '4e32fca'
```
Comment (coderanger) [inline Gemfile]: Locking it to a ref kind of defeats the point I would think?

Comment (thommay) [inline Gemfile]: nope; `--update` in the travis run causes it to update the vulnerabilities db.

**Comment by lamont-granquist** [comment]
LGTM, we'll see how annoying and/or useful it is :+1:

**Comment by smurawski** [comment]
:+1:

**Pull request merged**